tools/c7n_azure/c7n_azure/resources/key_vault.py

```diff
@@ -114,3 +119,48 @@ def enhance_policies(self, access_policies):
             policy['principalName'] = GraphHelper.get_principal_name(aad_object)
 
         return access_policies
+
+
+@resources.register('key')
+class KeyVaultKey(ArmResourceManager):
```

Comment:
Let's move it to its own file

Reply:
that seems a little extraneous .. modules as class groupings by functionality is quite normal in python.